<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWindow{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.008, y2:0.994, stop:0.0152381 rgba(5, 5, 122, 255), stop:1 rgba(186, 0, 216, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.008, y2:0.994, stop:0.0152381 rgba(5, 5, 122, 255), stop:1 rgba(186, 0, 216, 255));}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#tab_main,#tab_choose,#tab_workouthistory{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.008, y2:0.994, stop:0.0152381 rgba(5, 5, 122, 255), stop:1 rgba(186, 0, 216, 255));}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <widget class="QLabel" name="lbl_welcome">
        <property name="geometry">
         <rect>
          <x>635</x>
          <y>60</y>
          <width>650</width>
          <height>111</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_txt1">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>230</y>
          <width>321</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>You preformed so far:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_txt1_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>310</y>
          <width>331</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>You preformed so far:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_workouts_num">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>230</y>
          <width>321</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 170, 0);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>You preformed so far:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_avg">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>310</y>
          <width>331</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 170, 0);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>You preformed so far:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_word">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>420</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:20px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(205, 236, 255);</string>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_txt1_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>410</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>User Guide:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_choose">
       <attribute name="title">
        <string>Workout Estimation</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>735</x>
          <y>40</y>
          <width>450</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Choose exercise</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tb_ce">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>210</y>
          <width>950</width>
          <height>201</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Exercise name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Main Muscle Group</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Exercie id:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>420</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Reps number:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="i_eid">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_alert">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>470</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 0, 0)</string>
        </property>
        <property name="text">
         <string>Please Fill all the fields with numbers only!</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="i_repsnum">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>440</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_watch">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>530</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:20px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(205, 236, 255);</string>
        </property>
        <property name="text">
         <string>Watch instructions</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_start">
        <property name="geometry">
         <rect>
          <x>1230</x>
          <y>530</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:20px;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(205, 236, 255);</string>
        </property>
        <property name="text">
         <string>Start Workout</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_chosen">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>420</y>
          <width>551</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>txt</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_workouthistory">
       <attribute name="title">
        <string>My Workout History</string>
       </attribute>
       <widget class="QLabel" name="lbl_date3">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>480</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_score3">
        <property name="geometry">
         <rect>
          <x>1340</x>
          <y>150</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(248, 165, 0);
font: 75 36pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>A+</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_date2">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>480</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_score1">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>150</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(248, 165, 0);
font: 75 36pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>A+</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_name1">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>420</y>
          <width>261</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>exercise name</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_name3">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>420</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>exercise name</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_score2">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>150</y>
          <width>91</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(248, 165, 0);
font: 75 36pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>A+</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>580</y>
          <width>1091</width>
          <height>161</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Exercise name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reps</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="lbl_reps2">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>450</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>total reps</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>520</y>
          <width>521</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>More of your workouts:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>635</x>
          <y>30</y>
          <width>650</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 36pt &quot;MS Shell Dlg 2&quot;;color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>Your recently workouts</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_reps1">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>450</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>total reps</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_date1">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>480</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>date</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_name2">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>420</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>exercise name</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_reps3">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>450</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>total reps</string>
        </property>
       </widget>
       <widget class="QLabel" name="image1">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>250</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>IMAGE</string>
        </property>
       </widget>
       <widget class="QLabel" name="image2">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>250</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>IMAGE</string>
        </property>
       </widget>
       <widget class="QLabel" name="image3">
        <property name="geometry">
         <rect>
          <x>1300</x>
          <y>250</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string>IMAGE</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
